src/components/DeleteMember/DeleteMember.jsx

```diff
@@ -0,0 +1,63 @@
+import React, { PureComponent } from 'react';
+import { graphql } from 'react-apollo';
+import Button from 'material-ui/Button';
+import Dialog from 'material-ui/Dialog';
+import DialogActions from 'material-ui/Dialog/DialogActions';
+import DialogContent from 'material-ui/Dialog/DialogContent';
+import DialogContentText from 'material-ui/Dialog/DialogContentText';
+import DialogTitle from 'material-ui/Dialog/DialogTitle';
+
+import DeleteMemberMutation from './DeleteMemberMutation';
+
+class DeleteMember extends PureComponent {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      open: false
+    };
+  }
```

Comment:
You can assign the default state outside of the constructor for a bit more simple setup:
```javascript
class DeleteMember extends PureComponent {
  state = { open: false }
  // ...
}
```